gitlab/v4/cli.py

```diff
@@ -268,6 +270,9 @@ def _populate_sub_parser_by_class(
     if cls.__name__ in cli.custom_actions:
         name = cls.__name__
         for action_name in cli.custom_actions[name]:
+            if action_name in action_parsers:
+                continue
```

Comment:
As we discussed on Signal this seems to have a side effect and ignores overrides https://pastebin.com/sAGVNCxm

Example: https://github.com/python-gitlab/python-gitlab/blob/main/gitlab/v4/objects/files.py#L98-L101

I wonder if if having a `conflict_handler="resolve"` in our ArgumentParser would help here:
https://docs.python.org/3/library/argparse.html#conflict-handler

> Sometimes (e.g. when using [parents](https://docs.python.org/3/library/argparse.html#parents)) it may be useful to simply override any older arguments with the same option string. To get this behavior, the value 'resolve' can be supplied to the conflict_handler= argument of [ArgumentParser](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser):

Reply:
Thanks for the tip on `conflict_handler` that seems to solve the issue.